packages/sockethub/src/janitor.ts

```diff
@@ -57,6 +57,9 @@ function removeStaleSocketSessions(platformInstance, sockets) {
       rmLog('removing stale socket session reference ' + sessionId + ' in platform instance '
         + platformInstance.id);
       platformInstance.sessions.delete(sessionId);
+      for (let key in platformInstance.sessionCallbacks) {
+        platformInstance.sessionCallbacks[key].delete(sessionId);
+      }
```

Comment:
I think in modern JS, you could write this like so:

```js
      for (const sessionCallback of platformInstance.sessionCallbacks) {
        sessionCallback.delete(sessionId);
      }
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of

Reply:
This isn't the case for objects, you can't iterate over the values of them without some additional code.